.github/workflows/ci.yml

```diff
@@ -0,0 +1,275 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore: [ 'demos/**']
```

Comment:
Hi @sambua, this breaks some of the requirements from our existing continuous deployment workflow. Can you please remove this section and restore the deleted workflow file? It's better if we QA it before making changes to the existing solution.
```suggestion
```